aiven/resource_project_user_test.go

```diff
@@ -26,7 +25,7 @@ func TestAccAivenProjectUser_basic(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckAivenProjectUserAttributes("data.aiven_project_user.user"),
 					resource.TestCheckResourceAttr(resourceName, "project", fmt.Sprintf("test-acc-pr-%s", rName)),
-					resource.TestCheckResourceAttr(resourceName, "email", fmt.Sprintf("user%s@aiven.fi", rName)),
+					resource.TestCheckResourceAttr(resourceName, "email", fmt.Sprintf("savciuci-%s@aiven.fi", rName)),
```

Comment:
This email address doesn't look like it would work (minus instead of plus). In general would be good to get some actual test address for these.

Reply:
would love to use some actual test emails